packages/blockchain-wallet-v4-frontend/src/modals/CoinifyExchangeData/Create/AcceptTerms/index.js

```diff
@@ -18,7 +18,7 @@ class AcceptTermsContainer extends Component {
   }
 
   componentDidUpdate (prevProps) {
-    if (prevProps.signupError) {
+    if (prevProps.signupError||this.state.busy) {
```

Comment:
it should be `&&` here, otherwise it would still trigger when `this.state.busy` is falsy and `prevProps.signupError` is truthy.
Also spaces around operator are missing

Reply:
Whoops, got it.